.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: CodeCov
+
+on: [push, pull_request]
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ 3.7, 3.8, 3.9 ]  # latest release minus two
```

Comment:
Do we still need 3.7?

Reply:
Looks like all the major packages still support 3.7 so I guess it's fine

Reply:
Waiting on confirmation from the internal team on this.

Reply:
Looks like we commented at almost exactly the same time! - I agree, but will see what the guys come back with.